apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -2,76 +2,108 @@
 import { useIntl } from "react-intl";
 import { useQuery } from "urql";
 
-import { Pending } from "@gc-digital-talent/ui";
+import { Pending, Separator, ThrowNotFound } from "@gc-digital-talent/ui";
 import { ROLE_NAME } from "@gc-digital-talent/auth";
-import { User, graphql } from "@gc-digital-talent/graphql";
+import {
+  getFragment,
+  graphql,
+  ReviewApplicationPreviewListFragment,
+  ReviewRecruitmentProcessPreviewListFragment,
+} from "@gc-digital-talent/graphql";
 import { commonMessages } from "@gc-digital-talent/i18n";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
 
 import SEO from "~/components/SEO/SEO";
 import { getFullNameHtml } from "~/utils/nameUtils";
 import RequireAuth from "~/components/RequireAuth/RequireAuth";
 import Hero from "~/components/Hero";
+import { isQualifiedFinalDecision } from "~/utils/poolCandidate";
 
-export interface DashboardPageProps {
-  currentUser?: User | null;
-}
-
-export const DashboardPage = ({ currentUser }: DashboardPageProps) => {
-  const intl = useIntl();
-
-  return (
-    <>
-      <SEO title={""} description={""} />
-      <Hero
-        title={intl.formatMessage(
-          {
-            defaultMessage:
-              "Welcome back<hidden> to your applicant dashboard</hidden>, {name}",
-            id: "bw4CAS",
-            description:
-              "Title for applicant dashboard on the talent cloud admin portal.",
-          },
-          {
-            name: currentUser
-              ? getFullNameHtml(
-                  currentUser.firstName,
-                  currentUser.lastName,
-                  intl,
-                )
-              : intl.formatMessage(commonMessages.notAvailable),
-          },
-        )}
-        subtitle={""}
-      />
-    </>
-  );
-};
+import ReviewApplicationPreviewList, {
+  ReviewApplicationPreviewList_Fragment,
+} from "./components/ReviewApplicationPreviewList";
+import ReviewRecruitmentProcessPreviewList, {
+  ReviewRecruitmentProcessPreviewList_Fragment,
+} from "./components/ReviewRecruitmentProcessPreviewList";
 
 const ApplicantDashboard_Query = graphql(/* GraphQL */ `
-  query ApplicantDashboard_Query {
+  query ApplicantDashboard {
     me {
       id
       firstName
       lastName
+      poolCandidates {
+        ...ReviewApplicationPreviewList
+        ...ReviewRecruitmentProcessPreviewList
+      }
     }
   }
 `);
 
-export const ApplicantDashboardPageApi = () => {
+export const DashboardPage = () => {
+  const intl = useIntl();
+
   const [{ data, fetching, error }] = useQuery({
     query: ApplicantDashboard_Query,
   });
 
+  const user = data?.me;
+
+  const applications = getFragment<ReviewApplicationPreviewListFragment>(
```

Comment:
You do not need to strongly type this generic, passing the fragment as the first argument does this.

Reply:
Changed in [b7e4eb3](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/b7e4eb30269e46bbc3948d9cbf267c7c15a45111)

Reply:
Fixed in [988d3bb](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/988d3bb22efa2d2da6544b633783b3c70a3f7ab5)